common/main/java/com/couchbase/lite/FullTextIndexConfiguration.java

```diff
@@ -34,7 +34,7 @@ public FullTextIndexConfiguration(@NonNull String... expressions) {
         super(IndexType.FULL_TEXT, Arrays.asList(expressions));
     }
 
-    FullTextIndexConfiguration(@NonNull List<String> expressions) { super(IndexType.FULL_TEXT, expressions); }
+    public FullTextIndexConfiguration(@NonNull List<String> expressions) { super(IndexType.FULL_TEXT, expressions); }
```

Comment:
@pasin : client code needs to be able to create one of these?

Reply:
Yes